tests/conftest.py

```diff
@@ -65,3 +65,35 @@ def get_feats(rndstate):
     def _gen_feat(n_samples, n_features):
         return generate_feat(n_samples, n_features, rndstate)
     return _gen_feat
+
+
+def _get_linear_entanglement(n_qbits_in_block, feature_dim):
```

Comment:
For consistency in notations, you should use `n_features` instead of `feature_dim`.